notebook/templates/edit.html

```diff
@@ -1,39 +1,43 @@
-<!DOCTYPE html>
+<!doctype html>
 <html>
-<head>
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>{{page_config['appName'] | e}} - Edit</title>
-  {% block favicon %}
-  <link rel="icon" type="image/x-icon" href="{{ base_url | escape }}static/favicons/favicon-file.ico" class="favicon">
-  {% endblock %}
-</head>
-<body data-notebook="edit">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>{{page_config['appName'] | e}} - Edit</title>
+    {% block favicon %}
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="{{ base_url | escape }}static/favicons/favicon-file.ico"
+      class="favicon"
+    />
+    {% endblock %}
+  </head>
+  <body data-notebook="edit">
+    {# Copy so we do not modify the page_config with updates. #} {% set
```

Comment:
Looks like we might want to keep the previous line breaks here for readability.

Reply:
Done